src/ManageCourses.Api/Model/Course.cs

```diff
@@ -21,7 +21,9 @@ public class Course
         public string AccreditingProviderName { get; set; }
         public string Subjects { get; set; }
         public string StudyMode { get; set; }
-        public string AgeRange { get; set; }
+        public string AgeRange { get; set; }        
+        public CourseQualification Qualification { get; set; }
+        public string TypeDescription { get; set; }
```

Comment:
xmldoc with the kind of thing you'd expect to see in here

eg "qual(s) + full/part time + salary + apprenticeship"